reproduce.py

```diff
@@ -0,0 +1,138 @@
+import os
+import numpy as np
+import torch
+import shutil
+
+import train
+import eval
+
+train_params = {}
+
+"""
+save_base
+- Name of the directory to save models
+"""
+train_params["save_base"] = (
+    "./reproduce/"  # this will be the root directory of where we save models
+)
+
+"""
+experiment_name
+- Name of the directory where results for this run are saved.
+"""
+train_params["experiment_name"] = (
+    "repr"  # This will be the name of the directory where results for this run are saved.  # this is a subdirectory in save_base
+)
+
+
+"""
+species_set
+- Which set of species to train on.
+- Valid values: 'all', 'snt_birds'
+"""
+train_params["species_set"] = "all"  # unused parameter
```

Comment:
planning on deleting the unused parameters

Reply:
I think that would be the best!